src/http-launch.c

```diff
@@ -303,13 +303,19 @@ on_message (GstBus * bus, GstMessage * message, gpointer user_data)
       break;
     }
     case GST_MESSAGE_WARNING:{
-      gchar *debug;
-      GError *err;
+      gchar *debug = NULL;
+      GError *err = NULL;
 
       gst_message_parse_error (message, &err, &debug);
-      g_print ("Warning %s\n", err->message);
-      g_error_free (err);
-      g_free (debug);
+      //g_print ("Warning %s\n", err->message);
```

Comment:
The problem here is that err is NULL? That shouldn't happen at all and would be a bug somewhere in GStreamer.

Can you get me a debug log with GST_DEBUG=6 of the crash in question?